tests/cypress/tests/DuplicatePolicyInDiffNS.spec.js

```diff
@@ -35,8 +35,8 @@ describeT('RHACM4K-2342 - GRC UI: [P1][Sev1][policy-grc] Verify create policy wi
   })
   it('Delete created policies', () => {
     for (const policyName of policyNames) {
-       cy.actionPolicyActionInListing(policyName, 'Remove')
-       cy.verifyPolicyNotInListing(policyName)
+       cy.visit('/multicloud/policies/all').waitForPageContentLoad()
+       .actionPolicyActionInListing(policyName, 'Remove')
```

Comment:
Should we leave `.verifyPolicyNotInListing(policyName)` at the bottom? That seems like a good step to make sure the policy is no longer in the table...

Reply:
I don't think this step actual works because in this test case two policies are named same but in different namespace. I think I will move it after the loop to make sure neither of the policy presents

Reply:
Ah, good point! 🙂